res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -1405,6 +1412,53 @@ DJ505.CueLoopMode = function(deck, offset) {
 };
 DJ505.CueLoopMode.prototype = Object.create(components.ComponentContainer.prototype);
 
+DJ505.EditMode = function(deck, offset) {
+    components.ComponentContainer.call(this);
+    this.ledControl = DJ505.PadMode.HOTCUE;
+    this.color = DJ505.PadColor.ORANGE;
```

Comment:
Note to myself: Orange is for flip mode (and macros), I should use another color for this.

Reply:
I changed to color of the mode button to apple green instead.